CONTRIBUTING.md

```diff
@@ -4,6 +4,8 @@ All PRs that require modifications to the `README.md` or `values.schema.json` fi
 The `brand.sh` script should then be executed using the upstream properties, i.e. `./brand.sh infinispan.conf`, and all
 modified `*.tpl`, `*.md` and `*.json` files added to the commit.
 
+All PRs that require modifications to the `values.yaml` or `test/tls_values.yaml.yq` must run `./build_examples.sh` to rebuild all the examples.
```

Comment:
There's no `build_examples.sh` in the diff, is this outdated?

Reply:
yeah removing